README.md

```diff
@@ -100,11 +100,79 @@ this can be used with `watch`.
 watch -c juju status --color
 ```
 
-### Alternate deployment methods
+## Alternate deployment methods
 
+### Deploying storage
 
+The Canonical Distribution of Kubernetes allows you to connect with durable
+storage devices such as [Ceph](http://ceph.com). When paired with the
+[Juju Storage](https://jujucharms.com/docs/2.0/charms-storage) feature you
+can add durable storage easily and across all clouds.
 
-#### Usage with your own binaries
+Deploy the ceph-mon and ceph-osd charms.
+
+```
+juju deploy cs:ceph-mon -n 3
+juju deploy cs:ceph-osd -n 3
```

Comment:
I recall you guys mentioning at least one of these two needs to be a minimum of 3. Should we mention that? Also, do they both need to be a minimum of three, and/or be matching values? Not sure how much we want to educate people on ceph here, but we might consider the bare minimum of info to prevent user errors when the inexperienced try it out the first time.